safehttp/plugins/hsts/hsts.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hsts
+
+import (
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-safeweb/safehttp"
+)
+
+// Plugin implements automatic HSTS functionality.
+type Plugin struct {
+	// The time, in seconds, that the browser should remember
+	// that a site is only to be accessed using HTTPS.
+	MaxAge uint64
+
+	// This field controls the includeSubDomains directive.
```

Comment:
Doc should start with the name of the field it is documenting: "DisableInclude... disables ..."

same for all other fields. It is odd to see all godoc starting with "This field".